requirements/docs.txt

```diff
@@ -1,7 +1,7 @@
-nbconvert ~= 7.4.0
+curlify ~= 2.2.1
 Jinja2 ~= 3.1.2
 load-dotenv ~=0.1.0
-curlify ~= 2.2.1
-progressbar2 ~= 3.55.0
-sphinx ~= 5.3.0
-sphinx-rtd-theme ~= 1.2.0
+nbconvert ~= 7.10.0
+progressbar2 ~= 4.2.0
+sphinx ~= 7.1.2
```

Comment:
`sphinx` goes up to 7.2.6, but 7.2.0 onwards all require Python 3.9+. 7.1.2 was only released in August, and currently has no vulnerabilities.